wordpress-mobile/WordPress-Android
# Use View Binding in My Site

This PR introduces View Bindings for the Fragments and ViewHolders in the My Site section of the app. Most of the changes are pretty similar to what we've done so far in the other PRs. One exception is the `ImprovedMySiteFragment`, where it was necessary to resort to a couple of `findViewById`s (let me know if you have any suggestions for better alternatives).

Note: I've added a `@Suppress("ComplexMethod")` to the method that handles navigation events, as I can't think of a way to simplify it any further. Let me know if you any suggestions.

To test:

Smoke test the My Site screen and make sure everything is working as expected and nothing looks out of place. Some additional checks:

1. Make sure the account avatar is loaded correctly and that it correctly resizes on scroll.
1. Create a new site and make sure completing a Quick Start task correctly shows a Snackbar. Also try to pin/unpin/hide/remove a card and make sure it works as expected.
1. Enable/disable the `my_site_improvements_enabled` flag and repeat.

👉 I know the PR seems pretty big, but most of it is due to formatting. Selecting the "Hide whitespace changes" should make the review a lot easier.

PR submission checklist:
